File: hw/tipan/U2/Hw18_JavaScript_animal/animal.js/src/Food.js
const Animal = require('./Animal.js');
class Food {
    constructor(id , amount , type ) {
        this.id = id;
        this.amount = amount;
        this.type = type;
    }

    toString() {
        return `Food { id=${this.id}, amount=${this.amount}, type=${this.type} }`;
    }

  
    get id() {
      return this._id;
    }
  
    set id(value) {
      this._id = value;
    }
  
    get amount() {
      return this._amount;
    }
  
    set amount(value) {
      this._amount = value;
    }
  
    get type() {
      return this._type;
    }
  
    set type(value) {
      this._type = value;
    }
  
  }
  
   module.exports =Food;

File: ws/cuichan/u1/ws11DirectEngineering/AntSimulator/src/ec/edu/espe/antsimulator/model/Food.java
package ec.edu.espe.antsimulator.model;

/**
 *
 * @author David Cuichan
 */
public class Food {
    int amount;

    @Override
    public String toString() {
        return "Food{" + "amount=" + amount + '}';
    }

    public Food(int amount) {
        this.amount = amount;
    }
    
    
}
